mmdet3d/datasets/lyft_dataset.py

```diff
@@ -433,8 +433,7 @@ def json2csv(json_path, csv_savepath):
             json_path (str): Path of the result json file.
             csv_savepath (str): Path to save the csv file.
         """
-        with open(json_path, 'r') as f:
-            results = mmcv.load(f)['results']
+        results = mmcv.load(json_path)['results']
         csv_nopred = 'data/lyft/sample_submission.csv'
```

Comment:
1. We may use `osp.join(self.data_root, 'sample_submission.csv')`
2. csv_nopred -> sample_list_path